appinventor/components/src/com/google/appinventor/components/runtime/TextBoxBase.java

```diff
@@ -432,6 +436,28 @@ public void Hint(String hint) {
     view.invalidate();
   }
 
+  @SimpleProperty(
+          category = PropertyCategory.APPEARANCE,
+          description = "Specifies the color of the hint of the %type%.")
+  public int HintColor() {
+    return hintColor;
+  }
+
+  @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_COLOR, defaultValue = Component.DEFAULT_VALUE_COLOR_GRAY)
+  @SimpleProperty()
+  public void HintColor(int hintColor) {
+    this.hintColor = hintColor;
+    if (hintColor != 0xFF888888) {
+      view.setHintTextColor(hintColor);
+    } else {
+      if (isHighContrast || container.$form().HighContrast()) {
+        view.setHintTextColor(COLOR_YELLOW);
```

Comment:
Does yellow work in both light and dark themes?

Reply:
Yes.